test-case/multiple-pause-resume.sh

```diff
@@ -170,16 +170,17 @@ do
         echo
         if [ "$(pidof expect)" ]; then
             dloge "Still have expect process not finished after wait for $max_wait_time"
-            # now dump process
-            ps -ef |grep -E 'aplay|arecord' || true
+            # list aplay/arecord process and kill them
+            pgrep -a aplay && pkill -9 aplay
+            pgrep -a arecord && pkill -9 arecord
```

Comment:
This is really not just "fixing a trivial shellcheck warning"? This may or may not be a good change but it does not solve a problem I'm aware of and this PR is already pretty long so I would leave the `kill` out for now.

```suggestion
            pgrep -a -f arecord || true
```